### PR TITLE
Valued text input back to life

### PR DESCRIPTION
**Pull request by mahmoud-zakria-sociomantic**

### CODE REVIEW

File: src/DefaultTheme.js [hunk 1]
```diff
@@ -7,41 +7,42 @@
  *
  */
 
-import cx                         from 'classnames/bind';
+import cx                                               from 'classnames/bind';
```
Comment (daniel-martic-sociomantic) [inline src/DefaultTheme.js]: Maybe to keep this spacing short? Now `import` and `from` are far away and all lines are exceeding the length of 80 😕

File: src/ValuedTextInput/index.jsx [hunk 2]
```diff
@@ -91,156 +93,132 @@ ValuedTextInput.propTypes =
...
+    onKeyDown          : undefined,
+    onKeyPress         : undefined,
+    onKeyUp            : undefined,
+    onMouseOut         : undefined,
+    onMouseOver        : undefined,
+    placeholder        : undefined,
+    spellCheck         : undefined,
+    textAlign          : 'auto',
+    value              : undefined,
+    valueLabel         : '%',
```
Comment (daniel-martic-sociomantic) [inline src/ValuedTextInput/index.jsx]: I think this should remain `valueLabel: undefined`; `"valueLabel": "%"` is defined as an example in `defaults.json` (which is good), while this should be up to user to define (same as other `label` props)

Comment (mahmoud-zakria-sociomantic) [reply src/ValuedTextInput/index.jsx]: Well, when it's defined in `defaults.json` it doesn't work that's why I defined in both files to discuss about it.

**Comment by conor-cafferkey-sociomantic** [comment]
Oh, and also can you please go thru all the propTypes definitions and remove the unused ones

**Comment by conor-cafferkey-sociomantic** [comment]
Good to go

**Pull request merged**